docs/content/doc/developers/oauth2-provider.en-us.md

```diff
@@ -42,7 +42,41 @@ To use the Authorization Code Grant as a third party application it is required
 
 ## Scopes
 
-Currently Gitea does not support scopes (see [#4300](https://github.com/go-gitea/gitea/issues/4300)) and all third party applications will be granted access to all resources of the user and their organizations.
+Gitea supports the following scopes for tokens:
+
+| Name | Description |
+| ---- | ----------- |
+| **(no scope)** | Grants read-only access to public user profile and public repositories. |
+| **repo** | Full control over all repositories. |
+| &nbsp;&nbsp;&nbsp;&nbsp;**repo:status** | Grants read/write access to commit status in all repositories. |
```

Comment:
It's really necessary to include `&nbsp;`?

Reply:
Regular spaces will not be rendered visually on the web page, only `&nbsp;` will put an actual space.